examples/js-tests/parseInt/index.mjs

```diff
@@ -0,0 +1,6 @@
+import { loadStdlib } from "@reach-sh/stdlib";
```

Comment:
You should not make a new file. We want the tests to actually run and `js-tests` already runs